multinet/api/views/table.py

```diff
@@ -103,8 +108,14 @@ def get_rows(self, request, parent_lookup_workspace__name: str, name: str):
 
         pagination = ArangoPagination()
         query = ArangoQuery.from_collections(workspace.get_arango_db(), [table.name])
-        paginated_query = pagination.paginate_queryset(query, request)
 
+        # Attempt filtering
+        try:
+            query = query.filter(json.loads(request.query_params.get('filter', '{}')))
+        except json.JSONDecodeError:
+            pass
```

Comment:
What's going on here with swallowing the `JSONDecodeError`?

Reply:
If something besides JSON is passed into this parameter, we don't want this endpoint to crash, so we just ignore it.

Reply:
Why not? If someone passes malformed JSON by hand, we shouldn't just give them the filterless answer right?

Reply:
Good point. I was approaching this from the perspective of ensuring the endpoint doesn't throw a `500`, but we could return a `400` here, pointing out the `filter` parameter.

Reply:
55f9dd3